src/meta/src/hummock/compaction.rs

```diff
@@ -116,6 +125,15 @@ impl CompactStatus {
         let (prior, posterior) = (prior.last_mut().unwrap(), posterior.first_mut().unwrap());
         let is_select_level_leveling = matches!(prior, LevelHandler::Nonoverlapping(_, _));
         let is_target_level_leveling = matches!(posterior, LevelHandler::Nonoverlapping(_, _));
+        let ord_table_id = |x: u64, y: u64| {
+            if x == y {
+                Ordering::Equal
+            } else if y.wrapping_sub(x) <= u64::MAX / 2 {
```

Comment:
Why doesn't direct comparison fit here?

Reply:
Assume that earlier table_id x == u64::MAX and then later table_id y == 0

Reply:
> Why doesn't direct comparison fit here?

Assume that earlier table_id x == u64::MAX and then later table_id y == 0

Reply:
I think u64 for table_id should be large enough to avoid overflow but the wrapping_sub doesn't hurt anyway.